WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
This is a religious title and far more important to Meclchisedechians, in terms of Prestige and piety, than some petty duchy. It also makes it possible for King-tier Melchisedechians to hold it as their primary title.